### ISSUE TITLE
Resolve should clear selected conversation

### ISSUE THREAD

**Issue by Christilut** [chatwoot/chatwoot#4068]
**Is your enhancement request related to a problem? Please describe.**
When resolving a conversation, the conversation does not disappear. That is a bit strange in my opinion. I always press the "All Conversations" button after a resolve so my UI goes back to the initial state.

**Describe the solution you'd like**
Resolving a conversation should go back to "All Conversations". Or maybe the last view the user used.
But the important part is that the resolved conversation also disappears on the right side.

**Comment by pranavrajs** [comment]
@Christilut Are you suggesting that the action should be "Resolve and move to next" instead of just staying in the same view.

The current behaviour is built because the agent might need to add a note for internal team about the conversation after resolving it.

**Comment by Christilut** [comment]
Yeah moving to the next open conversation (or none if no other open conversation exists) would make the workflow nicer.
I think it's confusing and a bit weird UX that a resolved conversation would still be showing on the page that normally only shows open conversations.

The agent can add a note before closing it right?

**Comment by pranavrajs** [comment]
@Christilut 😄 Yeah, they could. This has been a constant debate for some time within the team. I believe a Resolve and move to next action along with resolve should work here.

**Comment by Christilut** [comment]
I think that would work yes. It would be very useful if the UI would remember the last used resolve action. So the big green button would be either `Resolve` or `Resolve and next` depending on which was last used.
